modules/creating-a-project-workbench.adoc

```diff
@@ -23,14 +23,18 @@ The *Data science projects* page opens.
 . Click the name of the project that you want to add the workbench to.
 +
 The *Details* page for the project opens.
-. Click *Create workbench* in the *Workbenches* section.
+. In the *Workbenches* section, click *Create workbench*.
 +
 The *Create workbench* page opens.
 . Configure the properties of the workbench you are creating.
-.. Enter a *name* for your workbench.
-.. Enter a *description* for your workbench.
-.. Select the *notebook image* to use for your workbench server.
-.. Select the *container size* for your server.
+.. In the *Name* field, enter a name for your workbench.
+.. Optional: In the *Description* field, enter a description to define your workbench.
+.. In the *Notebook image* section, complete the relevant fields to specify the most suitable notebook image to use with your workbench. 
+... From the *Image selection* list, select a notebook image from the list.
+.. In the *Deployment size* section, complete the relevant fields to specify the size of your deployment instance.
+... From the *Container size* list, select a suitable container size for your server.
+... Optional: From the *Accelerator* field, select an accelerator from the list. 
+... Optional: If you selected an accelerator in the preceding step, from the *Number of accelerators* list, select the number of accelerators to use.  
```

Comment:
Could trim this down to
'Optional: If you selected an accelerator in the preceding step, from the Number of accelerators list, select a value.'